src/createExplorer.js

```diff
@@ -173,7 +174,7 @@ class Explorer {
 
   loadPackageProp(filepath: string, content: string) {
     const parsedContent = loaders.loadJson(filepath, content);
-    const packagePropValue = parsedContent[this.config.packageProp];
+    const packagePropValue = get(parsedContent, this.config.packageProp);
```

Comment:
Not sure where to add the tests for the nested prop logic

Reply:
Tests will need to be added in all the places that `package.json` parsing is tested.

Reply:
@davidtheclark done!